index.html

```diff
@@ -0,0 +1,177 @@
+          <h3 class="program-card-title">Main Stage</h3>
+          <p class="program-card-description">Listen to the speakers from various countries about the messenges of
+            sharing and opening.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/business_center_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Business Land</h3>
+          <p class="program-card-description">In this land you will be able to explore new trends, acquire tools and
+            discuss innovative strategies to
+            expand your business.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/code_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Developer Land</h3>
+          <p class="program-card-description">In this scenario you will be able to discuss and discover the potential
+            of AI.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/sports_esports_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Gamer Land</h3>
+          <p class="program-card-description">Dive into the vast playing field this land has to offer and reveal its
+            secrets as you win a game to the
+            future.</p>
+        </article>
+        <article class="program-card">
+          <div class="program-card-icon">
+            <img src="./assets/keyboard_white_24dp.svg" alt="">
+          </div>
+          <h3 class="program-card-title">Workshops</h3>
+          <p class="program-card-description">Try creating your own work using open source license rather than just
+            watching at it.</p>
+        </article>
+      </div>
+      <a class="display display-none" href="#">SEE THE WHOLE PROGRAM</a>
+      <div class="join-button">
+        <p>Sign up to Jalisco Talent Land</p>
+      </div>
+    </section>
+
+    <section id="speakers" class="speakers">
+
+      <div id="flexcards" class="flex-card">
+      </div>
+
+      <div class="arrow_down">
+        <p>MORE</span></p>
+        <span><i class="fa-solid fa-angle-down fa-2x"></i>
+      </div>
+    </section>
+
+    <section class="container-fluid partner">
+      <div>
+        <h2>Partners</h2>
+        <hr>
+      </div>
+      <div>
+        <img src="./assets/HCLTech-new-logo.svg.png" alt="">
+        <img src="./assets/astrazeneca-logo-big.png" alt="">
+        <img src="./assets/2560px-Baxter.svg.png" alt="">
+        <img src="./assets/Logo_de_Steren.svg.png" alt="">
+        <img src="./assets/telcel-logo.svg" alt="">
+      </div>
+    </section>
+  </main>
+  <footer class="container-fluid footer-index">
+    <div>
+      <p class="footer_1">© Copyright 2023</p>
+    </div>
+    <div>
+      <p>Talent Network.</p>
```

Comment:
- [optional] for the desktop version, add space between the logo and text in the footer so it follows the original design